### ISSUE TITLE
Illegal Characters when sending coordinates

### ISSUE THREAD

**Issue by TylerS1066** [MamiyaOtaru/voxelmap#6]
Currently VoxelMap blindly takes the tab list and uses the format of `/msg {name} {coords}`.  I propose that you strip the name portion of colors because many servers will kick the player for illegal characters in chat when trying to send coordinates.

**Comment by MamiyaOtaru** [comment]
if it is not doing that it obviously should be.  Thanks for the heads up

**Comment by citylion** [comment]
Yeah this is causing a problem on the server I play on, I  hope it gets fixed soon, I'm a bit fan of voxelmap but I've unfortunately had to switch to another mapping mod for the time being.
